week-4/E-arrays-of-objects/exercise-1.js

```diff
@@ -25,11 +25,24 @@ WRITE YOUR CODE BELOW
 */
 
 
-var persons = // Complete here
+var persons = [person1, person2, person3];
+/// Complete here
 
-var personNames = // Complete here
+var personNames = persons.map(getNames);
 
-var personsYoungerThan28YearsOld = // Complete here
+function getNames (person){
+    return person.name;
+}
+
+// Complete here
+
+var personsYoungerThan28YearsOld = persons.filter(getYounger);
+
+function getYounger(person){
```

Comment:
I'd name this `isYoungerThan28` (or perhaps `isUnder28`).  This is a predicate function - it returns true or false when given an object to judge.   `get` prefixes are used for other methods which we'll meet later.